### PR TITLE
updated the portal api approval example

### PR DESCRIPTION
**Pull request by LLe27**
## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

**Review by sedkis — APPROVED** [review]

File: tyk-docs/content/tyk-apis/tyk-portal-api/portal-keys.md [hunk 1]
```diff
@@ -1614,6 +1614,9 @@ authorization:7a7b140f-2480-4d5a-4e78-24049e3ba7f8
 
 ```
 {
- Response here
+    "RawKey": "eyJvcmciOiI1ZTlkOTU0NGExZGNkNjAwMDFkMGVkMjAiLCJpZCI6ImQ0NzIzOWUxMjg3NTRjMGM5MTQ4MzYzMjg2YjhlZDQ2IiwiaCI6Im11cm11cjY0In0=",
+    "Password": ""
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-portal-api/portal-keys.md:1618]: is this the key you meant to put in the doc?
image

File: tyk-docs/content/tyk-apis/tyk-portal-api/portal-keys.md [hunk 2]
```diff
@@ -1614,6 +1614,9 @@ authorization:7a7b140f-2480-4d5a-4e78-24049e3ba7f8
...
 }
 ```
+
+**Note:** If you're using Authentication Token as your authentication mode, the `RawKey` value is the actual API Key used to send authenticated requests.
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-portal-api/portal-keys.md]: and if yo are not using authentication token?

Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-portal-api/portal-keys.md]:
```suggestion
**Note:** If you're using Authentication Token as your *authentication mode*, the `RawKey` value is the actual API Key used to send authenticated requests.
```

**Review by letzya — APPROVED** [review]

**Pull request merged**